governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -843,14 +843,111 @@ const Proposal = ({
 
 const Proposals = () => {
   const router = useRouter()
+  const [allProposals, setAllProposals] = useState<TransactionAccount[]>([])
   const [currentProposal, setCurrentProposal] = useState<TransactionAccount>()
+  const [currentProposalIndex, setCurrentProposalIndex] = useState<number>()
+  const [allProposalsVerifiedArr, setAllProposalsVerifiedArr] = useState<
+    boolean[]
+  >([])
   const [currentProposalPubkey, setCurrentProposalPubkey] = useState<string>()
+  const [allProposalsIxs, setAllProposalsIxs] = useState<
+    MultisigInstruction[][]
+  >([])
+  const [isAllProposalsIxsLoading, setIsAllProposalsIxsLoading] =
+    useState<boolean>(true)
+  const { cluster } = useContext(ClusterContext)
   const {
+    squads,
     priceFeedMultisigAccount,
     priceFeedMultisigProposals,
     isLoading: isMultisigLoading,
   } = useMultisigContext()
 
+  useEffect(() => {
+    const fetchAllProposalsInstructions = async () => {
+      if (squads) {
+        const allProposalsIxs = await getManyProposalsInstructions(
+          squads,
+          priceFeedMultisigProposals
+        )
+        const multisigParser = MultisigParser.fromCluster(
+          getMultisigCluster(cluster)
+        )
+        const parsedAllProposalsIxs = allProposalsIxs.map((ixs) =>
+          ixs.map((ix) =>
+            multisigParser.parseInstruction({
+              programId: ix.programId,
+              data: ix.data as Buffer,
+              keys: ix.keys as AccountMeta[],
+            })
+          )
+        )
+        const proposalsRes: TransactionAccount[] = []
+        const instructionsRes: MultisigInstruction[][] = []
+        // filter proposals for respective devnet/pythtest and mainnet-beta/pythnet clusters
+        parsedAllProposalsIxs.map((ixs, idx) => {
+          // pythtest/pythnet proposals
+          if (
+            isRemoteCluster(cluster) &&
+            ixs.length > 0 &&
+            ixs.every((ix) => ix instanceof WormholeMultisigInstruction)
```

Comment:
If you want to be precise maybe something more like : 
`(ix instanceof WormholeMultisigInstruction) && (ix.name == "postMessage") && (ix.governanceAction instanceof ExecutePostedVaa ) &&  (ix.governanceAction.targetChainId === 'pythnet')`

Reply:
the purpose of this was to show all Wormhole proposals and those that are not verified will be easily distinguishable through the verified checks

Reply:
I you want to show all the proposals (it's fine) you need to come up with better logic. 
Right now, if a proposal has both a WormholeInstruction and a PythInstruction it won't show anywhere

Reply:
is that possible though?